Dockerfile

```diff
@@ -37,8 +40,8 @@ ENV HELM_PLUGINS /opt/helm-plugins
 
 COPY --from=builder /workspace/manager .
 COPY --from=builder /workspace/helm-plugins ${HELM_PLUGINS}
-
-COPY build/charts/ /charts/
+COPY --from=builder /workspace/build/charts /
```

Comment:
Don't we need `charts` instead of `build/charts` here @yevgeny-shnaidman?

Reply:
After talking to @yevgeny-shnaidman `build/charts` contains also the charts themselves + index files and tar.gz so it is OK to get `build/charts` instead of `charts/`